.github/workflows/auth-e2e-tests.yml

```diff
@@ -23,43 +37,44 @@ jobs:
           check-latest: true
       - run: npm ci
 
-      - name: 'Create env file'
+      - name: Setup Postgres Database
+        run: |
+          npm run db:setup --workspace apps/institution-service
+
+      - name: "Create env file"
         run: |
           ENV_FILE_PATH=./apps/authentication-service/.env
+          INST_ENV_FILE_PATH=./apps/institution-service/.env
           touch ${ENV_FILE_PATH}
-          
+          touch ${INST_ENV_FILE_PATH}
+
           # Vars
           echo -e "${{ needs.setup-env.outputs.env_vars }}" >> ${ENV_FILE_PATH}
-          
+
           # Secrets (can't load these from another job, due to GH security features)
           echo AUTH0_CLIENT_SECRET=${{ secrets.AUTH0_CLIENT_SECRET }} >> ${ENV_FILE_PATH}          
           echo E2E_CLIENT_SECRET=${{ secrets.E2E_CLIENT_SECRET }} >> ${ENV_FILE_PATH}
           echo AUTH_PASSWORD_WITH_KEY_ROLES=${{ secrets.AUTH_PASSWORD_WITH_KEY_ROLES }} >> ${ENV_FILE_PATH}
           echo AUTH_PASSWORD_WITHOUT_KEY_ROLES=${{ secrets.AUTH_PASSWORD_WITHOUT_KEY_ROLES }} >> ${ENV_FILE_PATH}
 
-          cat ${ENV_FILE_PATH}
+          # Secrets for institution list
+          echo WIDGET_CLIENT_SECRET=${{ secrets.WIDGET_CLIENT_SECRET }} >> ${ENV_FILE_PATH}
+          echo WEB_UI_CLIENT_SECRET=${{ secrets.WEB_UI_CLIENT_SECRET }} >> ${ENV_FILE_PATH}
+          echo E2E_INSTITUTION_PASSWORD=${{ secrets.E2E_INSTITUTION_PASSWORD }} >> ${ENV_FILE_PATH}
+
+          # Copy the env file to the institution service
+          cp ${ENV_FILE_PATH} ${INST_ENV_FILE_PATH} 
 
       - name: Start Institution Service
-        run: npm run dev --workspace apps/institution-service
+        run: npm run dev --workspace apps/institution-service &
```

Comment:
Without the `&`, the tests do not continue. GH thinks the command hasn't finished, so the workflow gets stuck, and never finishes.